backend/flindt/feedback/models.py

```diff
@@ -47,7 +47,8 @@ class Remark(FlindtBaseModel):
     rating = models.ForeignKey(
         Rating,
         related_name='rating',
-        null=True
+        null=True,
+        on_delete=models.deletion.CASCADE
```

Comment:
I think you can do `models.CASCADE` which is a bit shorter so you can keep a lot of these changes on a single line.